languages.json

```diff
@@ -1,5 +1,13 @@
 {
   "languages": {
+    "_8th": {
```

Comment:
I think it would be better to rename this, and move it into the E section
```suggestion
    "Eighth": {
```

Reply:
Good idea!